protocol/0085-RVST-rewards_vesting.md

```diff
@@ -29,6 +30,35 @@ $$T = max(B_{vesting} * r * a, m)$$
 
 When transferring funds from the vesting account to the vested account, a new transfer type should be used, `TRANSFER_TYPE_REWARDS_VESTED`.
 
+## Vested bonus
+
+Once vested rewards are transferred to the vested account, the party will be able to transfer funds to their general account using a normal transfer.
+
+Alternatively, they can leave their rewards in the vested account to receive a multiplier on their reward payout share. The size of this multiplier is dependent on the balance (expressed in quantum) in the vested account before rewards are distributed.
```

Comment:
This should be based on the total in the vested account, the unvested account, and the earned rewards not yet transferred into the unvested account.